src/components/Carousel.js

```diff
@@ -285,8 +285,7 @@ class Carousel extends Component {
         const itemSize = isHorizontal ? firstItem.clientWidth : firstItem.clientHeight;
 
         this.setState((_state, props) => ({
-            itemSize: itemSize,
-            wrapperSize: isHorizontal ? itemSize * Children.count(props.children) : itemSize
```

Comment:
Why this line (289) was removed?

Reply:
wrapperSize is never read anywhere in the component, and is reported as such here: https://lgtm.com/projects/g/leandrowd/react-responsive-carousel/snapshot/dist-1791630058-1548143554293/files/src/components/Carousel.js?sort=name&dir=ASC&mode=list#L14